modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc

```diff
@@ -43,10 +43,16 @@ You can use any of the following options for authentication and authorization wh
 
 ** Using <<create-api-ASM-CLI, user credentials>> (username and password)
 +
-This option supports only those users that are managed by Anypoint Platform.
+This option supports only those users who:
+
+*** Are managed by Anypoint Platform
+*** Have not enabled multi-factor authentication
```

Comment:
It is not the user that enables MFA, it is the organization. I would change it to something like:
`*** Are not enrolled in multi-factor authentication`

Reply:
Same below

Reply:
Disregard my previous comment